ShopAPI/ShopAPI/Models/Product.cs

```diff
@@ -27,6 +29,47 @@ public class Cpu : Product
         public string Series { get; set; }
         public bool IntegratedGraphics { get; set; }
     }
+    
+    public class CpuCooler : Product
+    {
+       public string Socket { get; set; } 
+       public bool IsWaterCooled { get; set; }
+       public string Size { get; set; } // set as string for now because sizes are usually represented as LxWxH
+    }
+
+    public class Motherboard : Product
+    {
+        public string Socket { get; set; }
+        public string Chipset { get; set; }
+        public string MemoryType { get; set; }
+        public string FormFactor { get; set; }
+    }
+
+    public class Memory : Product
+    {
+        public string MemoryType { get; set; }
+        public int Speed { get; set; }
+        public int Size { get; set; }
+    }
+
+    public class Storage : Product
```

Comment:
Lets add a type for storage (ssd vs hdd)

Reply:
Done